gtk4-macros/src/lib.rs

```diff
@@ -8,7 +8,7 @@ use proc_macro::TokenStream;
 use proc_macro_error::proc_macro_error;
 use syn::{parse_macro_input, DeriveInput};
 
-#[proc_macro_derive(CompositeTemplate, attributes(template_child))]
+#[proc_macro_derive(CompositeTemplate, attributes(template, template_child))]
```

Comment:
This macro seriously needs some docs :)

Reply:
Added a first draft... help is more than welcome :)

Reply:
The version of the docs you pushed looks good to me :) @sdroege ?

Reply:
Seems all fine to me